src/util/util.c

```diff
@@ -41,7 +41,7 @@ int metres_to_feet(int metres)
 
 int mph_to_kmph(int mph)
 {
-	// 1 mph = 1.60934 kmph
+	// 1 mph = 1.60934 kph
```

Comment:
kph is colloquial, but not correct. kmph is correct but not very pretty. km/h is the most common (except for the UK) and also correct.